KubeArmor/packaging/post-install.sh

```diff
@@ -19,3 +19,7 @@ fi
 # start kubearmor.service
 /bin/systemctl daemon-reload
 /bin/systemctl start kubearmor.service
+
+# Set default GRPC listening port for kubearmor as environment variable
+echo "KUBEARMOR_SERVICE=:32767" >> /etc/environment
+. /etc/environment
```

Comment:
sourcing of this environment file won't have effect on the env outside of the script. User still has to logout/login for changes to come into effect.

Reply:
Thanks a lot. I have updated the script